lib/ui/employee_id/employee_id_card.dart

```diff
@@ -472,14 +463,14 @@ class _EmployeeIdCardState extends State<EmployeeIdCard> {
     }
   }
 
-  double letterSpacingForTablet() {
+  double letterSpacingForTablet(BuildContext context) {
     if (MediaQuery.of(context).orientation == Orientation.landscape) {
       return ScalingUtility.horizontalSafeBlock * 1;
     }
     return ScalingUtility.horizontalSafeBlock * 3;
   }
 
-  double fontSizeForTablet() {
+  double fontSizeForTablet(BuildContext context) {
```

Comment:
Functions that use BuildContext do not need to pass it as a parameter. Use the context instance variable from the class itself.